mne/viz/epochs.py

```diff
@@ -922,10 +935,16 @@ def plot_epochs_psd(epochs, fmin=0, fmax=np.inf, tmin=None, tmax=None,
         Figure distributing one image per channel across sensor topography.
     """
     from .raw import _set_psd_plot_params, _convert_psds
+
+    if spatial_colors is None:
```

Comment:
I would also suggest to use doc template to make sure the docstring here match raw.plot_psd

Reply:
@larsoner Ok, changed default to `None`

@agramfort If I understand you correctly, the new parameters were already copied directly from the docstring in raw, cf. viz/Epochs.py 920-928. Is something still missing?

Reply:
@jshanna what @agramfort means is that instead of copy-pasting a parameter like `spatial_colors : ...` what you should do is:
```
    %(spatial_colors)s
```
and then modify `mne/utils/docs.py` to have a `spatial_colors` entry in the `docdict`. See how we do it for example for `random_state` or any of the other `docdict` entries.